core/game_object.lua

```diff
@@ -1986,6 +2003,70 @@ Set `prefix_config.key = false` on your object instead.]]):format(obj.key), obj.
         inject = function(self) end
     }
 
+    SMODS.PokerHandPart {
+        key = '_highest',
+        func = function(hand) return get_highest(hand) end
+    }
+    SMODS.PokerHandPart {
+        key = '_straight',
+        func = function(hand) return get_straight(hand) end
+    }
+    SMODS.PokerHandPart {
+        key = '_flush',
+        func = function(hand) return get_flush(hand) end,
+    }
+    -- all sets of 2 or more cards of same rank
+    SMODS.PokerHandPart {
+        key = '_fh',
```

Comment:
A bit misleading to call this _fh if it's not necessarily a full house

Edit: yeah, if it's called _fh people might use it when it's just a pair. something like all_pairs or something might be better

Reply:
Renamed it to `_all_pairs`